### ISSUE TITLE
bug: if metadata isn't sent json sets it to null

### ISSUE THREAD

**Issue by 0xJepsen** [harnesslabs/arbiter#696]
# Problem
meta data serialized incorrectly when set to None
Set it to an empty object

**Comment by Autoparallel** [comment]
I don't know where this error is. Can you give more detail?

**Comment by 0xJepsen** [comment]
Yeah let me try to remember. I think it was in the was the data collection was serializing the metadata.
